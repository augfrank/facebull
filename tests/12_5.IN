M0   C0   C2  2300
M1   C0   C3  2746
M2   C0   C6  2525
M3   C0   C7  2063
M4   C0   C9  2052
M5   C0   C10  2162
M6   C1   C0  2906
M7   C1   C2  2723
M8   C1   C3  2221
M9   C1   C4  2349
M10   C1   C5  2966
M11   C1   C6  2860
M12   C1   C8  2045
M13   C1   C10  2207
M14   C1   C11  2097
M15   C2   C1  2765
M16   C2   C3  2646
M17   C2   C4  2227
M18   C2   C6  2656
M19   C2   C7  2453
M20   C2   C10  2600
M21   C2   C11  2364
M22   C3   C2  2308
M23   C3   C8  2596
M24   C3   C9  2534
M25   C4   C1  2939
M26   C4   C3  2092
M27   C4   C5  2335
M28   C4   C6  2361
M29   C4   C8  2529
M30   C4   C9  2328
M31   C4   C10  2861
M32   C5   C0  2305
M33   C5   C3  2080
M34   C5   C9  2320
M35   C5   C10  2367
M36   C5   C11  2665
M37   C6   C1  2533
M38   C6   C2  2574
M39   C6   C4  2524
M40   C6   C7  2917
M41   C6   C8  2070
M42   C6   C10  2526
M43   C6   C11  2541
M44   C7   C0  2258
M45   C7   C1  2390
M46   C7   C2  2197
M47   C7   C4  2495
M48   C7   C8  2241
M49   C7   C9  2813
M50   C7   C10  2052
M51   C7   C11  2527
M52   C8   C0  2185
M53   C8   C3  2681
M54   C8   C4  2886
M55   C8   C7  2965
M56   C8   C9  2762
M57   C8   C10  2343
M58   C8   C11  2452
M59   C9   C1  2064
M60   C9   C2  2229
M61   C9   C3  2931
M62   C9   C5  2712
M63   C9   C6  2950
M64   C9   C10  2658
M65   C10   C1  2885
M66   C10   C2  2134
M67   C10   C3  2808
M68   C10   C4  2248
M69   C10   C5  2654
M70   C10   C6  2411
M71   C10   C9  2934
M72   C11   C0  2560
M73   C11   C1  2010
M74   C11   C3  2506
M75   C11   C6  2054
M76   C11   C7  2646
M77   C11   C9  2795
