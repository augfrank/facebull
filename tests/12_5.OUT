25895
3 14 17 24 27 33 41 50 52 59 66 75
